<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentEditorView</class>
 <widget class="QWidget" name="DocumentEditorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addDocButton">
       <property name="text">
        <string>New Document</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadDocButton">
       <property name="text">
        <string>Load Document</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Remove Document</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
